Content.Server/Lube/LubeSystem.cs

```diff
@@ -31,28 +32,30 @@ public override void Initialize()
         SubscribeLocalEvent<LubeComponent, GetVerbsEvent<UtilityVerb>>(OnUtilityVerb);
     }
 
-    private void OnInteract(EntityUid uid, LubeComponent component, ref AfterInteractEvent args)
+    private void OnInteract(Entity<LubeComponent> entity, ref AfterInteractEvent args)
     {
         if (args.Handled)
             return;
 
         if (!args.CanReach || args.Target is not { Valid: true } target)
             return;
 
-        if (TryLube(uid, component, target, args.User))
+        if (TryLube(entity, entity, target, args.User))
```

Comment:
I think I understand the reason for wanting to go back to the old format :D

But the correct thing to do would be to change the TryGlue function to use Entity<>

in any case, you are under no obligation to do so in this PR

Reply:
No no, this was just me running on auto-pilot and didn't think to consider fixing the TryLube/Glue functions too. I'll do that now, there's no reason for me not to :)